roles/elastic-stack/ansible-elasticsearch/tasks/Debian.yml

```diff
@@ -26,7 +26,7 @@
 
 - name: Install Oracle Java 8
   become: true
-  apt: name=openjdk-8-jdk state=latest
+  apt: name=openjdk-8-jdk
```

Comment:
Why is the `state=latest` statement not needed anymore?

Reply:
Hi @manuasir,

This change came as result to Ansible-Linting error:

` Package installs should not use latest ` reported in this issue: https://github.com/ansible/ansible-lint/issues/479

This change was made in this commit: https://github.com/ansible/ansible-lint/commit/f5fcee4eb3bea623e625de2ede02a3e2686e5e1e#diff-42252eea388ddb7d54414f584e9ebc78.

Sincerely, and like many other users facing this issue, I think this restriction has no logic due to the need to upgrade a package to its latest version without worrying about specifying a determined one. 

Anyway, I think `apt` will take the last version by default, so no worries.

Kind regards,

Rshad